client/src/views/Report/Sunburst/index.js

```diff
@@ -0,0 +1,223 @@
+import React, { useMemo, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    PieChart,
+    Pie,
+    Cell,
+    Tooltip,
+    ResponsiveContainer,
+} from 'recharts';
+
+import styles from './styles.scss';
+
+const getSunBurstData = (dataObject) => {
+    const centerPieData = [{
+        key: dataObject.key,
+        name: dataObject.name,
+        size: dataObject.size,
+    }];
+
+    const firstLevelData = [];
+    const secondLevelData = [];
+
+    dataObject.children.forEach(({
+        key,
+        name,
+        size,
+        children,
+    }) => {
+        firstLevelData.push({
+            key,
+            name,
+            size,
+        });
+
+        secondLevelData.push(children.map(
+            child => ({
+                ...child,
+                firstLevelKey: key,
+            }),
+        ));
+    });
+
+    return {
+        centerPieData,
+        firstLevelData,
+        secondLevelData: secondLevelData.flat(),
+    };
+};
+
+const getFilteredData = (newlevel, newKey, dataValue) => {
+    if (newlevel === 'first') {
+        const filteredata = dataValue.children.filter(dvc => dvc.key === newKey);
+        return {
+            centerPieData: [{
+                key: dataValue.key,
+                name: dataValue.name,
+                size: dataValue.size,
+            }],
+            firstLevelData: [{
+                key: filteredata[0].key,
+                name: filteredata[0].name,
+                size: filteredata[0].size,
+            }],
+            secondLevelData: filteredata[0].children,
+        };
+    }
+    if (newlevel === 'second') {
+        const tempData = getSunBurstData(dataValue);
+
+        const slData = tempData.secondLevelData.find(fld => fld.key === newKey);
+
+        const flData = tempData.firstLevelData.filter(fld => fld.key === slData.firstLevelKey);
+
+        return {
+            centerPieData: [],
+            firstLevelData: flData,
+            secondLevelData: [slData],
+        };
+    }
+    return getSunBurstData(dataValue);
+};
+
+const getCellColor = (dataKey, colorSchemeValue) => {
+    if (dataKey.includes('Primary')) {
+        return colorSchemeValue[0];
+    }
+    return colorSchemeValue[1];
```

Comment:
retur 
```suggestion
    return colorSchemeValue[dataKey.includes('Primary') ? 0 : 1];
```

Reply:
We will be using '#rscz/SunBurst'. So sunburst removed !!